tests/test_bgp_af.rb

```diff
@@ -132,44 +137,41 @@ def setup
-    [:next_hop_route_map,            :ios_xr, 'red',     %w(ipv4 unicast),     :CliError],
-    [:next_hop_route_map,            :ios_xr, 'red',     %w(ipv6 unicast),     :CliError],
-    [:next_hop_route_map,            :ios_xr, 'red',     %w(ipv4 multicast),   :CliError],
-    [:next_hop_route_map,            :ios_xr, 'red',     %w(ipv6 multicast),   :CliError],
+    [:client_to_client,              :ios_xr, 'red',     :any,                 :CliError],
+    [:additional_paths_send,         :ios_xr, :any,      :multicast,           :CliError],
+    [:additional_paths_receive,      :ios_xr, :any,      :multicast,           :CliError],
+    [:additional_paths_selection,    :ios_xr, :any,      :multicast,           :CliError],
+    [:next_hop_route_map,            :ios_xr, 'red',     :any,                 :CliError],
     [:maximum_paths,                 :ios_xr, :any,      %w(l2vpn evpn),       :CliError],
     [:maximum_paths_ibgp,            :ios_xr, :any,      %w(l2vpn evpn),       :CliError],
-    [:additional_paths_selection,    :ios_xr, :any,      %w(ipv4 multicast),   :CliError],
-    [:additional_paths_selection,    :ios_xr, :any,      %w(ipv6 multicast),   :CliError],
+    [:route_target_import,           :ios_xr, 'default', :any,                 :CliError],
+    [:route_target_export,           :ios_xr, 'default', :any,                 :CliError],
 
     # Nexus Unsupported
-    [:any,                           :nexus,  :any,      %w(vpnv4 multicast),  :unsupported],
-    [:any,                           :nexus,  :any,      %w(vpnv6 multicast),  :unsupported],
 
     # Nexus CLI Errors
     [:any,                           :nexus,  'default', %w(l2vpn evpn),       :CliError],
-    [:additional_paths_install,      :nexus,  :any,      %w(ipv6 unicast),     :CliError],
-    [:additional_paths_install,      :nexus,  :any,      %w(ipv6 multicast),   :CliError],
-    [:advertise_l2vpn_evpn,          :nexus,  'default', %w(ipv4 unicast),     :CliError],
-    [:advertise_l2vpn_evpn,          :nexus,  'default', %w(ipv6 unicast),     :CliError],
-    [:advertise_l2vpn_evpn,          :nexus,  'default', %w(ipv4 multicast),   :CliError],
-    [:advertise_l2vpn_evpn,          :nexus,  'default', %w(ipv6 multicast),   :CliError],
-    [:advertise_l2vpn_evpn,          :nexus,  'red',     %w(ipv4 multicast),   :CliError],
-    [:advertise_l2vpn_evpn,          :nexus,  'red',     %w(ipv6 multicast),   :CliError],
+    [:additional_paths_install,      :nexus,  :any,      :ipv6,                :CliError],
+    [:advertise_l2vpn_evpn,          :nexus,  'default', :any,                 :CliError],
+    [:advertise_l2vpn_evpn,          :nexus,  'red',     :multicast,           :CliError],
+    [:inject_map,                    :nexus,  :any,      :multicast,           :CliError],
+    [:route_target_both_auto,        :nexus,  :any,      :any,                 :CliError],
```

Comment:
So the idea with here plus the above is that the route-target commands are only supported in a VRF on Nexus?